downloadphp.ps1

```diff
@@ -1,5 +1,31 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-Invoke-WebRequest -Uri https://windows.php.net/downloads/releases/php-7.2.7-nts-Win32-VC15-x64.zip -OutFile php.zip
-Expand-Archive -LiteralPath php.zip -DestinationPath php\
+$workingdir = (Get-Location).Path
+if($PSVersiontable.PSVersion.Major -lt 3)
+{
+    Write-Warning "Please download php and place it in the same folder as this script."
+    Write-Output "Download from: https://windows.php.net/downloads/releases/php-7.2.7-nts-Win32-VC15-x64.zip"
+    Write-Output ("Save to this directory: $workingdir `nand rename it to php.zip" -f (Get-Location).Path)
+    Read-Host "Press Enter when you're done!"
+}
+else
+{
+    Invoke-WebRequest -Uri https://windows.php.net/downloads/releases/php-7.2.7-nts-Win32-VC15-x64.zip -OutFile php.zip
+}
+if($PSVersiontable.PSVersion.Major -lt 5)
+{
+    if((Test-Path "$workdingdir\php") -eq $False)
+    {
+        Add-Type -AssemblyName System.IO.Compression.FileSystem
+        [IO.Compression.ZipFile]::ExtractToDirectory('php.zip', 'php\')
```

Comment:
Exception calling "ExtractToDirectory" with "2" argument(s): "Could not find file 'C:\WINDOWS\system32\php.zip'."